Makefile.inc

```diff
@@ -4,12 +4,14 @@ deploytool ?= operator
 registry_inmemory ?= true
 
 SCRIPTS_DIR ?= /opt/shipyard/scripts
+CLUSTERS_ARGS ?= ''
+DEPLOY_ARGS ?= ''
```

Comment:
These aren’t needed, although I suppose they play some use in documenting the variables (but we should really document the build template separately...)

Reply:
OK I thought its best to declare them rather than use potentially undeclared variables in the calls, but I get your point.
Would you rather I remove them or add comments above them that indicate their use?

Reply:
I don’t like clutter, I’d vote for removing them ;-).